duckduckgo/iOS
# apply new indicator assets

Reviewer: Mia or Caine
Asana: https://app.asana.com/0/414235014887631/489057161850493
CC:

**Description**:

The letter in the indicator graphics were too high by 1 px and this was distracting to some users.

**Steps to test this PR**:

1. Visit sites with a range of privacy grades and ensure that site rating letter looks centered in the circle.


###### Reviewer Checklist:


###### PR DRI Checklist:
